counterparty-core/counterpartycore/lib/messages/fairmint.py

```diff
@@ -37,6 +38,15 @@ def validate(
 
     asset_supply = ledger.supplies.asset_supply(db, fairminter["asset"])
 
+    if fairminter["max_mint_per_address"] is not None and fairminter["max_mint_per_address"] > 0:
+        alread_minted = ledger.issuances.get_fairmint_by_address(db, fairminter["tx_hash"], source)
+        if fairminter["price"] > 0:
+            if alread_minted + quantity > fairminter["max_mint_per_address"]:
+                problems.append("quantity exceeds maximum allowed by address")
```

Comment:
I think it'd be good to resolve all of these test coverage warnings? esp. for this fiddly protocol code

Reply:
I'm 100% sure this line is covered: https://github.com/CounterpartyXCP/counterparty-core/pull/3070/files#diff-a451bad1f5439bad46428f051f59e526a60a2be5d7e027e5612083c407fe8082R49-R60
I was careful to cover 100% of the lines added or modified.
See https://github.com/CounterpartyXCP/counterparty-core/pull/3070/checks?check_run_id=38654957041